src/apify/_configuration.py

```diff
@@ -353,6 +356,11 @@ class Configuration(CrawleeConfiguration):
         ),
     ] = None
 
+    @classmethod
+    def get_global_configuration(cls) -> Configuration:
+        """Retrieve the global instance of the configuration.
 
-# Monkey-patch the base class so that it works with the extended configuration
-CrawleeConfiguration.get_global_configuration = Configuration.get_global_configuration  # type: ignore[method-assign]
```

Comment:
I don't think we can just remove this - existing code may rely on `crawlee.Configuration.get_global_configuration` returning `apify.Configuration`.

Reply:
Reverting